updated calling rust from frontend example (#784)
tauri-apps/tauri-docs: docs/guides/features/command.md

> // With the Tauri API npm package:

Reviewer: ```suggestion
// When using the Tauri API npm package:
```

import { invoke } from '@tauri-apps/api/tauri'

> // With the Tauri global script, enabled when `tauri.conf.json > build > withGlobalTauri` is set to true:

Reviewer: ```suggestion
// When using the Tauri global script (if not using the npm package
// Be sure to set `build.withGlobalTauri` in `tauri.conf.json` to true
```

> const invokeTauri = window.__TAURI__.invoke

Reviewer: ```suggestion
const invoke = window.__TAURI__.invoke
```
